photutils/psf/models.py

```diff
@@ -6,6 +6,15 @@
 
 import numpy as np
 
+try:
+    import scipy
+    HAS_SCIPY = True
+except ImportError:
+    HAS_SCIPY = False
+
+if HAS_SCIPY:
+    from scipy.interpolate import RectBivariateSpline
+
```

Comment:
@mcara  This code here (lines 9-17) is unnecessary.  For optional dependencies like `scipy`, simply add the line `from scipy.interpolate import RectBivariateSpline` within the function or class where it is needed.  If one then tries to use the specific function/class, an `ImportError` will be raised at that time (i.e. not when `photutils` is imported).